src/FNA3D_Driver_Vulkan.c

```diff
@@ -94,6 +94,7 @@ static const uint32_t deviceExtensionCount = SDL_arraysize(deviceExtensionNames)
 #define FAST_TEXTURE_STAGING_SIZE 64000000 /* 64MB */
 #define STARTING_SLOW_TEXTURE_STAGING_SIZE 16000000 /* 16MB */
 #define MAX_SLOW_TEXTURE_STAGING_SIZE 256000000 /* 256MB */
+#define DEVICE_LOCAL_HEAP_USAGE_FACTOR 0.8
```

Comment:
This might need to be a value you can set externally (via an env var or whatever)

Reply:
With the new defrag work we might actually be able to turn this off entirely - I imagine OS programs using Vulkan will know to fall back to host memory as well.

Reply:
I'm fine with leaving this in as a debug env var if people want to simulate running their games on low-memory devices, but I agree that we can probably just bump this up to 1.0 at the very least now.